### PR TITLE
added navbar, form, and buttons

### PR DESCRIPTION
**Pull request by stephen-ho**

### CODE REVIEW

File: styles/Header.module.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.title a,
+.title span {
+  color: #0070f3;
+  text-decoration: none;
+}
+.title a:hover,
+.title a:focus,
+.title a:active {
+  text-decoration: underline;
+}
```
Comment (phil-ho) [inline styles/Header.module.css]: do these styles do anything right now?

File: styles/Nav.module.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+}
+.nav ul {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style: none;
+}
+.nav ul li a {
+  margin: 5px 15px;
+}
```
Comment (phil-ho) [inline styles/Nav.module.css]: Nit: its better to create classes for the `ul` and `a` elements than to style through the tag names

**Review by phil-ho — CHANGES_REQUESTED** [review]
a couple style fixes but overall looks good! thanks

File: styles/Header.module.css (hunk 1 above)
Comment (stephen-ho) [reply styles/Header.module.css]: thanks i'll remove the a's

File: styles/Nav.module.css (hunk 2 above)
Comment (stephen-ho) [reply styles/Nav.module.css]: Ok fixed

**Review by phil-ho — APPROVED** [review]

**Pull request merged**